src/sentry/static/sentry/app/components/activity/note/index.jsx

```diff
@@ -50,9 +60,9 @@ class Note extends React.Component {
   };
 
   handleDelete = () => {
-    const {item, onDelete} = this.props;
+    const {onDelete} = this.props;
 
-    onDelete(item);
+    onDelete(this.props);
```

Comment:
Is this intentional? We probably don't want to call this with *all* of the props

Reply:
Yeah, this component doesn’t need to make a decision on what the call handler needs.